### ISSUE TITLE
Issue facets: unreported missing values

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1189]
**Issue facets: unreported missing values**

Records of the datasets from the "RECOLNAT" publisher miss content in important data, like Country, Coordinates, and Collectiondate. The reporting e.g. for `https://www.gbif.org/occurrence/search?offset=80&dataset_key=b4b8935c-9ede-43ad-9326-84065b36c927` only included minor taxonomic issues, but none of these rather large ones.

The reason for Collection Date issues missing is possibly that this item is missing from the facet altogether (other dates are there, but not this one) (?)

-----

User provided contact info: @ahahn-gbif
System: Firefox 58.0.0 / Windows 10 0.0.0
User: [See in registry](https://www.gbif.org/api/feedback/user/6f95d550d57dac8cf86a810154719cef:f2d0c9ce0442c6f20cae31ffe0b1701bae9f7b5da0aa1bbef77c57ae40818e496542f6b6fc63050045c1f19356b3c49318fa1a4f8b5755cce54951bd6b5871c9)
Referer: https://www.gbif.org/occurrence/search?offset=80&dataset_key=b4b8935c-9ede-43ad-9326-84065b36c927
Window size: width 1457 - height 820
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-02-21T14:43:36.780Z',mode:absolute,to:'2018-02-21T14:49:36.780Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-02-21T14:43:36.780Z',mode:absolute,to:'2018-02-21T14:49:36.780Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
Collection Date is not something that is in the API I see
https://www.gbif.org/developer/occurrence

**Comment by MortenHofft** [comment]
I don't believe we consider missing coordinates nor missing country an issue do we?

**Comment by ahahn-gbif** [comment]
Sorry, I mean: eventDate (occurrence). Or am I overlooking that and it has a different label?
Coordinates / country: if they are missing altogether in occurrence data, I think we should at least flag that - after all we are flagging a whole lot of other things, too,  that are way less important than location.

**Comment by MortenHofft** [comment]
Event date is there but not exactly easy to find. it is in the advanced section
https://www.gbif.org/occurrence/search?event_date=2017-01-03&advanced=1

**Comment by MortenHofft** [comment]
missing coordinates. this is essentially what this does
https://www.gbif.org/occurrence/search?has_coordinate=false&advanced=1

**Comment by MortenHofft** [comment]
for unknown country we have
https://www.gbif.org/occurrence/search?country=ZZ&advanced=1
but I'm not sure that is properly populated. @MattBlissett will know

**Comment by MortenHofft** [comment]
But neither of these are considered flags. I agree it is natural to look for them there. It would make sense to add them as far as I can see

**Comment by ahahn-gbif** [comment]
Speaking for those who would like to point at these reports for highlighting the issues most valuable to fix when contacting a publisher, I would say that including those in the set of flags would make a lot of sense.

**Comment by MattBlissett** [comment]
Unknown country / ZZ isn't added to records, since it's not necessarily true. There may be a text description, or a country like "USSR" etc.

Negative searches is the overriding feature which would solve a lot of this (where country = null etc).
